Dual Battery Support.md

```diff
@@ -0,0 +1,262 @@
+#  Dual Battery Support
+
+macOS has buggy dual battery support and we won't have correct battery status without doing some changes.
+Many laptops do have dual batteries, one common example are Lenovo ThinkPad Laptops.
+In this quick guide we will explain on how to implement dual battery support under macOS.
+
+**Note:** this quick guide only covers the dual battery implementation part and not the patching process for battery reporting.
+
+## Requirements
+
+- [MaciASL](https://github.com/acidanthera/MaciASL)
+- Dump of native DSDT
+- Plist Editor (use your prefered one)
+- [SSDT-BATC](https://github.com/acidanthera/VirtualSMC/blob/master/Docs/SSDT-BATC.dsl)
+- Completed Sample (`BATC-Sample.dsl` and `BATC-Sample.plist`) for reference.
+
+## Implementation
+
+As an example for this quick guide we will use the Lenovo ThinkPad T440S laptop.
+In order to have correct battery status, in a dual battery scenario, we need to use SSDT-BATC which combines two batteries into a single one and aside from the SSDT itself, we also need to rename the notifiers of BAT0 & BAT1 to BATC
+
+Notifiers can be in short and long versions, some examples are:
+
+```
+Notify (BAT0, 0x80)
+Notify (BAT1, 0x81)
+Notify (\_SB.PCI0.LPC.EC.BAT0, 0x01)
+Notify (\_SB.PCI0.LPCB.EC.BAT1, 0x03)
+```
+
+In order for our SSDT-BATC to work, we should rename these notifiers to BATC, in our example:
```

Comment:
Renaming is when you change the name of a device (e. g. HECI to IMEI) and all references to it. Here we're changing references from individual batteries to a combined one, so we should say *change* instead of *rename*.

Reply:
Thanks for the hint, ok will do so in next upcoming commit.

Reply:
Resolved in latest commit